go.mod

```diff
@@ -5,11 +5,9 @@ go 1.21
 require (
 	github.com/alexliesenfeld/health v0.8.0
 	github.com/bradleyfalzon/ghinstallation/v2 v2.10.0
+	github.com/cenkalti/backoff v2.2.1+incompatible
```

Comment:
The doc site shows `v2.2.1+incompatible`(From 2019?) as latest but their repo have v3.X.X and v4.X.X
Can we use the newer one?

Reply:
Updated to the latest one